src/components/Workspace.js

```diff
@@ -10,19 +10,26 @@ import ns from '../config/css-ns';
  * @memberof Workspace
  * @private
  */
-const Workspace = ({ windows }) => (
-  <div className={ns('workspace')}>
-    {
-      Object.values(windows).map(window => (
-        <ConnectedWindow
-          id={window.id}
```

Comment:
The only thing that confuses me about this PR is the removal of `id` from `Window`. Can anyone provide insight into that?

Reply:
It's because `Window` gets the `id` from the actual `window` state. I think this was a redundant attribute, even before the PR. See here: https://github.com/ProjectMirador/mirador/blob/master/src/components/Window.js#L48

Reply:
Ah! Many thanks for the context, @jbaiter. That resolves my concern.